content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -57,7 +57,23 @@ kubectl proxy
 
 Kubectl will make Dashboard available at http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/.
 
-The UI can _only_ be accessed from the machine where the command is executed. See `kubectl proxy --help` for more options.
+
+To make kubernetes dashboard accessible remote using SSH tunnel and configure proxy:
+
+Example:
+
+```
+ssh -L 8001:127.0.0.1:8001 user@remote
+
+kubectl proxy --address='0.0.0.0' --accept-hosts='^*$'
```

Comment:
This is not something that we would advise. It will work, but you shouldn't serve on 0.0.0.0 and accept all hosts.